packages/eas-cli/src/commandUtils/context/contextUtils/getProjectIdAsync.ts

```diff
@@ -101,6 +101,17 @@ export async function getProjectIdAsync(
       );
     }
 
+    const actorUsername = getActorUsername(actor);
+    if (!exp.owner && appForProjectId.ownerAccount.name !== actorUsername) {
+      throw new Error(
+        `Project config: Project identified by "extra.eas.projectId" (${
```

Comment:
It's not clear based on the message or a link what the user should do. Maybe add sth like "You need to specify owner field"?